graylog2-server/src/main/java/org/graylog/security/authservice/AuthenticationDetails.java

```diff
@@ -32,6 +32,8 @@ public static Builder builder() {
         return Builder.create();
     }
 
+    public abstract AuthenticationDetails.Builder toBuilder();
```

Comment:
It's nice to have this, but we don't have code calling this. If we decide to keep the PR for the UI changes, we should leave this change in.